### ISSUE TITLE
colombia frequency plan AU915 or US915

### ISSUE THREAD

**Issue by chivoX** [helium/miner#751]
Hey guys I see on the list that Colombia shows as it should use AU915, but I've seen ppl buy US915 miners. 
This is a recurring question on the official Helium discord for Colombia.

Should we get AU915's or US915's ? What happens with ppl that got the wrong one ? will they still be able to mine HNT ?

this is even more confusing:
AU915
https://github.com/helium/miner/blob/master/priv/countries_reg_domains.csv#L48

US915
https://github.com/helium/devdocs/blob/master/longfi/frequency-plans-by-country.md#c-

thanks, and let me know if this is not the right place to raise this concern

**Comment by shawaj** [comment]
AU915 is correct as per the official lora alliance document here https://lora-alliance.org/resource_hub/rp2-1-0-3-lorawan-regional-parameters/

**Comment by mariocortes1987** [comment]
@chivoX I bougth SENSECAP M1 LORAWAN US915. Does it work in Colombia?

**Comment by sp193** [comment]
LoRaWAN uses ISM bands. You should consult your local regulations to determine if a given plan will fit. Given that TTN chose AU915 was over US915 for Colombia, it might be that the whole US915 range isn't supported. But I believe it is only a recommendation and you should just ensure the device complies with your local regulations.

The major difference between US915 and AU915 is the range for uplink channels. US915 might use 903.2 to 914.9MHz, while AU915 uses 915.2 to 927.8MHz. Both use 923.3 to 927.5MHz for downlinks. I say "might" because not all devices have support for all channels, but might instead use sub-band(s) - groups of 8 channels.
Basically, US915 uses a larger frequency range, as the uplink & downlink channels do not overlap.

**Comment by mariocortes1987** [comment]
Thank you for your Answer. I was reading technical documentation and the device support US915 and AU 915
